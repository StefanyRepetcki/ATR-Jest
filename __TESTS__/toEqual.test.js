// https://jestjs.io/pt-BR/docs/expect#toequalvalue

const Person = require('../modulo-1/aula2/index.js');

const stefany = new Person(
    "Stefany Repetcki",
    "stefany@hotmail.com",
    new Date(1999, 8, 23)
);

const objTest = new Person(
    "Stefany Repetcki",
    "stefany@hotmail.com",
    new Date(1999, 8, 23)
);

test("should check if the object stefany equal the properties of object objTest", () => {
    expect(stefany).toEqual(objTest);
    expect(stefany).toMatchSnapshot();
});