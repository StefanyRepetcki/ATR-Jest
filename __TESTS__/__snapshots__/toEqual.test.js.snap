// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should check if the object stefany equal the properties of object objTest 1`] = `
Person {
  "birthDay": 1999-09-23T03:00:00.000Z,
  "email": "stefany@hotmail.com",
  "fullName": "Stefany Repetcki",
}
`;
